iotdb/src/main/java/org/apache/iotdb/db/metadata/MGraph.java

```diff
@@ -366,8 +381,19 @@ public MeasurementSchema getSchemaForOnePathWithCheck(String path) throws PathEr
   @Override
   public String toString() {
     StringBuilder sb = new StringBuilder();
-    sb.append("===  Timeseries Tree  ===\n\n");
+    sb.append(TIME_SERIES_TREE_HEADER);
     sb.append(mtree.toString());
     return sb.toString();
   }
+
+  /**
+   * combine multiple metadata in string format
+   */
+  public static String combineMetadataInStrings(String[] metadatas) {
+    for (int i = 0; i < metadatas.length; i++) {
+      metadatas[i] = metadatas[i].replace(TIME_SERIES_TREE_HEADER, "");
```

Comment:
Hm.... I think, `TIME_SERIES_TREE_HEADER`  should not be add in `MGraph`, but in `TsServiceImpl`.
```
case "METADATA_IN_JSON":
        String metadataInJson = null;
        try {
          metadataInJson = MManager.getInstance().getMetadataInString();
        } catch (OutOfMemoryError outOfMemoryError) { // TODO OOME
          LOGGER.error("Failed to fetch all metadata in json", outOfMemoryError);
          status = getErrorStatus(String.format("Failed to fetch all metadata in json because: %s", outOfMemoryError));
          resp.setStatus(status);
          return resp;
        }
        resp.setMetadataInJson(metadataInJson);
        status = new TS_Status(TS_StatusCode.SUCCESS_STATUS);
        break;
```

Maybe modification like this can not avoid `metadatas[i].replace(TIME_SERIES_TREE_HEADER, "")`, but it makes the MGraph.toString() worthy of the javadoc.
```
  /**
   * functions for converting the mTree to a readable string in json format.
   */
```

Reply:
I think this should not be resolved here, because this have to change the TSServiceImpl, which has been changed in cluster branch, this could cause conflicts when we merge cluster into master.

Reply:
> I think this should not be resolved here, because this have to change the TSServiceImpl, which has been changed in cluster branch, this could cause conflicts when we merge cluster into master.

It is not a reasonable explanation,  so all modifications on TSServiceImple must be blocked until your cluster branch is finished??

Reply:
Fixed, but I remove this to MManager, not TSServiceImpl, because when I combine multiple JSON metadata, all the redundant TSHeaders are removed in MManager, so I think the adding and removing should happen in the same place, which is MManager. Moreover, this update does not conflicts with the javadoc of MManager.getMetadataInString().
`
/**
   * function for getting metadata in string.
   */
`